e2e/cypress/tests/support/api/system.d.ts

```diff
@@ -97,7 +97,7 @@ declare namespace Cypress {
          *       // do something with config
          *   });
          */
-        apiUpdateConfig(newConfig: AdminConfig): Chainable<AdminConfig>;
+        apiUpdateConfig(newConfig: AdminConfig): Chainable<{config: AdminConfig}>;
```

Comment:
Please revert change made here and update to latest master. In master, it's now using `DeepPartial`.